### PR TITLE
Process content from post request

### LINKED ISSUE #235: openwhisk emulator should pass request body to pipeline
If there is a `body.content` in the `actionParams`, add it to the `payload` (like openwhisk would). Depends on https://github.com/adobe/helix-simulator/issues/179.

### PR DESCRIPTION
**Pull request by rofe**
Fixes #235

### CODE REVIEW

File: src/utils/openwhisk.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ function extractActionContext(params) {
     __ow_headers,
     __ow_method,
     __ow_logger,
+    content,
```
Comment (tripodsan) [inline src/utils/openwhisk.js]: I'm not sure if we should remove it. we don't remove the other params neither....

In any case: does't lint complain that there is an unused variable? maybe better to delete it:

```
delete disclosed.content;
```

**Comment by tripodsan** [comment]
please  remove `package-lock.json` from your PR...

File: src/utils/openwhisk.js (hunk 1 above)
Comment (rofe) [reply src/utils/openwhisk.js]: I think `content` is a special case compared to other params. lint didn't complain, but I'll `delete` it anyway.

**Pull request merged**